crates/apollo-mcp-server/src/apps.rs

```diff
@@ -276,6 +279,8 @@ struct Manifest {
     resource: String,
     name: Option<String>,
     description: Option<String>,
+    #[serde(default)]
```

Comment:
nit: Is this necessary? I thought Serde deserializes optional fields to None by default.